### DISCUSSION TITLE
[SOLVED] Express Plugins Nuxt 3 RC

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#16361] [Questions]
Nevermind. Nuxt is frontend only. I got mislead.

**Comment by wobsoriano** [comment]
lol wut

**Reply by angelhdzmultimedia** [reply]
> lol wut

lol so funny i am rolling on the floor laughing!

**Comment by TheAlexLichter** [comment]
Nuxt is **not** frontend-only 👀

**Reply by angelhdzmultimedia** [reply]
I know. I just didn't want to debate with the "geniuses" who argued me against it being also backend. By the way, I solved my problem and started the express server with express-session and Steam Strategy for Passport.js.
and it worked like a charm.

Cheers!
